tirocks/src/table/plain.rs

```diff
@@ -0,0 +1,97 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+use tirocks_sys::rocksdb_PlainTableOptions;
+
+pub type EncodingType = tirocks_sys::rocksdb_EncodingType;
+
+#[repr(transparent)]
+pub struct PlainTableOptions(rocksdb_PlainTableOptions);
+
+impl PlainTableOptions {
+    /// plain table has optimization for fix-sized keys, which can
+    /// be specified via user_key_len.  Alternatively, you can pass
+    /// `kPlainTableVariableLength` if your keys have variable
+    /// lengths.
+    #[inline]
+    pub fn set_user_key_len(&mut self, len: u32) -> &mut Self {
```

Comment:
why not written in `simple_access` manner

Reply:
Because it doesn't need C function.